### ISSUE TITLE
Wrapped stables alignment?

### ISSUE THREAD

**Issue by szist** [input-output-hk/Bridges-working-group#15]
As more and more bridges are appearing to ETH chains there is a risk that the wrapped token space gets too fragmented: multiple versions of wrapped stables. Each of these wrapped tokens would be "owned" by a specific bridge. Until actual trusted stables appear on cardano it might cause very slow adoption after the nomad hack. Cardano users could probably be highly skeptical towards wrapped stables. Is there a way where major bridges would come together and reduce the risk for users to increase adoption and trust?

**Comment by jacktnho** [comment]
I agree that there should have some standards / frameworks to be set for stablecoins (or wrapped stablecoins) on Cardano. Probably those standards can be accomplished with some CIPs, which are expected to be in another working group.

My feeling is, there seems to be a big question mark still on whether the Cardano native token and the minting policy can fulfil the needs of those prominent stablecoins.
